src/library/Uno.Material/Styles/Controls/Chip.xaml

```diff
@@ -0,0 +1,571 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:not_win="http://uno.ui/not_win"
+					xmlns:toolkit="using:Uno.UI.Toolkit"
+					mc:Ignorable="not_win">
+
+	<x:String x:Key="ClearGlyphPathData">M10.661012,7.5689991L7.5990001,10.650999 12.939089,15.997999 7.5990001,21.336999 10.661012,24.405 16.007082,19.065 21.369997,24.405 24.430058,21.336999 24.429081,21.336 19.088991,15.998999 24.429081,10.662001 21.345095,7.5819996 16.007082,12.919001z M15.997072,0C24.828983,0 31.994999,7.1770013 31.994999,15.999998 31.994999,24.826997 24.828007,31.999999 15.997072,31.999999 7.1569835,31.999999 1.5270052E-07,24.826997 0,15.999998 1.5270052E-07,7.1799997 7.1569835,0 15.997072,0z</x:String>
+	<x:String x:Key="CheckGlyphPathStyle">M28.718018,0L32,3.2819897 10.666016,24.616999 0,13.951997 3.2810059,10.670007 10.666016,18.055033z</x:String>
+
+	<x:Double x:Key="ChipHeight">32</x:Double>
+	<x:Double x:Key="ChipElevation">8</x:Double>
+	<x:Double x:Key="CheckGlyphHeight">12</x:Double>
+	<CornerRadius x:Key="ChipCornerRadius">16</CornerRadius>
+
+	<x:Double x:Key="ThumbnailSize">24</x:Double>
+	<CornerRadius x:Key="ThumbnailCornerRadius">12</CornerRadius>
+
+	<Style x:Name="DeleteButtonStyle"
+		   TargetType="Button">
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="Button">
+					<Grid x:Name="ButtonLayoutGrid"
+						  Background="Transparent">
+
+						<VisualStateManager.VisualStateGroups>
```

Comment:
We don't have a pressed state for this?

Reply:
Apparently not. I've actually copied this from what we have in textbox. There's actually another question I wanted to ask (when this PR is opened) about this. Should I moved this in button.xaml and reference it when needed instead of copying this style twice?

Reply:
Also, yes I can make sure we have a pressing

Reply:
Basically I've got inspired by what wpf https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit, which they add a darker border brush on hover/pressed